src/context/ApiContext.tsx

```diff
@@ -404,7 +404,7 @@ export const ApiProvider: React.FC = ({ children }) => {
         publishSubCourse,
         joinBBBmeeting,
         getMentoringMaterial,
-        getFeedbackCallData,
+        getPeerToPeerCallData: getFeedbackCallData,
```

Comment:
In order to maintain the naming consistent, _all_ variable names should be renamed. If this turns out to be too tedious it would also be fine to me just to stick to "FeedbackCall" concerning variable names and only change the displayed texts. ;D

Reply:
@lsc0498 i changed now the naming to be consistent across all files, where there is feedback between students and pupils, but didn't change the variable names that has to do with the feedbak about the website

Reply:
@lsc0498 only file that i was not sure about was the variables in file called Steps_feedback
and there is a problem with the husky pre-commit, where husky says that a variable was declared but never used, when it's in fact used

 would appreciate your feedback on these 2 issues :)